### ISSUE TITLE
Can I change the directory and filename by editing the additional parameters?

### ISSUE THREAD

**Issue by jedmarcnocum** [aaccurso/Canvas2ImagePlugin#1]
document.getElementById('myCanvas'),
        '/path/to/directory',
        'filename'

So i can do a custom path and filename for saving my canvas?

**Comment by aaccurso** [comment]
Yes, that's exactly why I made a fork of the original plugin, to add that feature :D

**Comment by jedmarcnocum** [comment]
@aaccurso this is a great idea! But it seems like i cant make it to work, and when you install it using the cordova CLI it still gives the default plugin without filepath and filename parameters, so what i did was to replace the .js file and the .java file in the folder plugin, but its not working for me, when i tried to save the canvas the file path and file plugin did not change.

**Comment by aaccurso** [comment]
@jedmarcnocum so here is the deal, I won't be maintaining this plugin anymore, but I'll gladly merge a pull request if you make it work :D
